registration-processor/init/registration-processor-packet-receiver-stage/src/main/java/io/mosip/registration/processor/packet/receiver/exception/handler/PacketReceiverExceptionHandler.java

```diff
@@ -176,6 +177,12 @@ private PacketReceiverResponseDTO handleInvalidTokenException(InvalidTokenExcept
 		return buildPacketReceiverExceptionResponse((Exception)e);
 	}
 
+	private PacketReceiverResponseDTO handleInvalidTokenException(AuthenticationException e) {
+		InvalidTokenException ex = new InvalidTokenException(e.getMessage());
+		regProcLogger.error(LoggerFileConstant.SESSIONID.toString(),LoggerFileConstant.APPLICATIONID.toString(), ex.getErrorCode(),  e.getStackTrace()[0].toString());
+		return buildPacketReceiverExceptionResponse((Exception)ex);

```

Comment:
This method returns the entire stackstrace? It should return a simple error and message not the stackstrace.